staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -300,11 +299,23 @@ func (c *Config) ApplyClientCert(clientCAFile string) (*Config, error) {
 
 type completedConfig struct {
 	*Config
+
+	//===========================================================================
```

Comment:
Should delete the newline here? These three commented lines should belong to `*Config`, right?

Reply:
It's header for everything following.